packages/@lwc/babel-plugin-component/src/__tests__/observed-fields.spec.js

```diff
@@ -78,6 +78,59 @@ describe('observed fields', () => {
         }
     );
 
+    pluginTest(
+        'should not add reserved words to fields when is a reserved word decorated with @api, @track or @wire',
```

Comment:
suggestion > `should not process reserved class names decorated with @api, @track or @wire as fields `